scripts/scripts.js

```diff
@@ -138,7 +139,7 @@ const { loadArea, setConfig } = await import(`${miloLibs}/utils/utils.js`);
   decorateFigure();
   decorateVideo();
   decorateQuote();
-  decorateContent();
+  await decorateContent();
```

Comment:
Why this change?

Reply:
the lint tool was giving a warning as `decorateContent` is `async`